services/post_service.py

```
@@ -0,0 +1,85 @@
+            raise HTTPException(status_code=status.HTTP_403_FORBIDDEN,
+                                detail="해당 게시글에 대한 수정/삭제 권한이 없습니다.")
+
+    async def create_post(self,req:PostsCreateRequest, author_id : str) -> dict:
+        post_data = req.model_dump()
+
+        post_data["author_id"] = author_id
+        post_data["created_at"] = datetime.now(timezone.utc).isoformat()
+        post_data["views"] = 0
+        post_data["likes"] = 0
+        post_data["comment_count"]=0
+        saved_post = self.post_repo.save(post_data)
```

Ruff (0.14.11)

Local variable `saved_post` is assigned to but never used

Remove assignment to unused variable `saved_post`

(F841)